.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
 version: 2.1
 
 defaults:
-  rust_image: &rust_image quay.io/tarilabs/rust_tari-build-with-deps:nightly-2021-05-09
+  rust_image: &rust_image quay.io/tarilabs/rust_tari-build-with-deps:nightly-2021-08-17
```

Comment:
I will build and push this version to quay.io as it doesn't exist yet

Reply:
Ah... the problem with the integration tests. Thanks.